### PR TITLE
Added error page, Fixed login functionality, Fixed search functionality, Added read JSON functionality

### PR DESCRIPTION
**Pull request by dionwu110**
Added an error page when the user tries to enter a page that does not exist or if they try to enter session.html when a session_id is not selected/exists.

image

Added read JSON functionality and sample session data to be displayed onto the database table.
Also truncated the session ID and user ID so that not many characters are displayed.

image

Fixed search functionality and allows user to search through database by table header instead of general search of all table cells.

image

image

I'd like to apologize in advance for the very big commit! 😅

### CODE REVIEW

File: database.html [hunk 1]
```diff
@@ -75,10 +73,78 @@ <h1>MyMood Database Table</h1>
...
+      }
+      zebraStripe();
+    }
+    </script>
+    
+    <!-- Read JSON File -->
+    <script>
+    $(document).ready(function readJson() {
+      $.getJSON("/assets/json/sessions.json", function(data) {
+        // Declare variables
```
Comment (lydarren) [inline database.html]: same as above

File: database.html [hunk 2]
```diff
@@ -75,10 +73,78 @@ <h1>MyMood Database Table</h1>
...
+      $evenRows.removeClass('odd').addClass('even');
+    };
+    </script>
+    
     <!-- Search Function -->
     <script>
-      search.init(["#database_table"]);
-      search.searchRows();
+    function search() {
+      // Declare variables
```
Comment (lydarren) [inline database.html]: "// Declare variables" is an obvious comment making it redundant.
For variable names, whats tr and td? I'm guessing tr means tableRow but I'm not sure about td.

File: session.html [hunk 3]
```diff
@@ -74,10 +68,70 @@ <h1>Viewing Session #11223344:</h1>
...
+      $evenRows.removeClass('odd').addClass('even');
+    };
+    </script>
+    
     <!-- Search Function -->
     <script>
-      search.init(["#session_table"]);
-      search.searchRows();
+    function search() {
+      // Declare variables
```
Comment (lydarren) [inline session.html]: same as above

File: database.html (hunk 1 above)
Comment (dionwu110) [reply database.html]: I am assuming that the tr tag means tableRow, and then the td tag means tableData which represents a cell on the table. So then I'd be adding cells to the table row by row.

**Comment by dionwu110** [comment]
I've discussed this topic with Daniel in person and we thought it'd probably be too complicated to put it in a separate file to be reusable. So then I didn't try to implement it. However, it was a lot more simpler than I thought and was able to implement it. I will add another commit to add the file.

**Pull request merged**